internal/pkg/auth/auth.go

```diff
@@ -146,6 +146,18 @@ func addOrUpdateContext(config *v1.Config, isCloud bool, credentials *Credential
 		}
 	}
 
+	stateSalt, err := secret.GenerateRandomBytes(saltLength)
+	if err != nil {
+		return err
+	}
+	stateNonce, err := secret.GenerateRandomBytes(nonceLength)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	salt, err := secret.GenerateRandomBytes(saltLength)
	if err != nil {
		return err
	}
	nonce, err := secret.GenerateRandomBytes(nonceLength)
	if err != nil {
		return err
	}
```

Reply:
two pairs of salt and nonce are being generated in this function so i want to name them differently